src/components/ui/Checkbox/Checkbox.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
Need more test cases:
1. Need to test that when you click on the checkbox, its value changes to `checked`
2. Test if it renders left border given `withBorder` prop